.github/workflows/aws.yml

```diff
@@ -61,68 +54,127 @@ jobs:
             #### Terraform Plan 📖\`${{ steps.plan.outcome }}\`
+
             <details><summary>Show Plan</summary>
+
             \`\`\`\n
             ${process.env.PLAN}
             \`\`\`
+
             </details>
+
             *Pusher: @${{ github.actor }}, Action: \`${{ github.event_name }}\`*`;
+
             github.issues.createComment({
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
               body: output
             })
+
       - name: Terraform Plan Status
-        working-directory: src/test/resources/terraform
         if: steps.plan.outcome == 'failure'
         run: exit 1
 
       - name: Terraform Apply
-        working-directory: src/test/resources/terraform
         if: github.ref == 'refs/heads/main' && github.event_name == 'push'
         run: terraform apply -auto-approve
 
-      - name: Set Database Host URL
-        working-directory: src/test/resources/terraform
-        run: echo "DBHOST=$(terraform output -json | jq -r '.dbHost.value')" >> $GITHUB_ENV
-
-      - name: Create Database Objects
-        working-directory: src/test/resources/terraform
-        run: PGPASSWORD=LbRootPass1 psql -v ON_ERROR_STOP=1 --username lbuser --dbname lbcat -h $DBHOST -f postgres-init.sql
-
   test:
     runs-on: ubuntu-18.04
     needs: [setup]
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - database: postgres
+            version: 9-6
+          - database: postgres
+            version: 10
+          - database: postgres
+            version: 11
+          - database: postgres
+            version: 12
+          - database: postgres
+            version: 13          
+
     steps:
       - uses: actions/checkout@v2
 
-      - name: Update Harness Config
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+          terraform_wrapper: false
+          
+      - name: Terraform Init
+        working-directory: src/test/resources/terraform
+        run: terraform init
+
+      - name: Get Public IP
+        id: ip
+        uses: haythem/public-ip@v1.2
+
+      - name: Print Public IP to tfvars file
+        working-directory: src/test/resources/terraform
+        run: |
+          echo 'public_ip = "${{ steps.ip.outputs.ipv4 }}"' >> github.auto.tfvars
+
+      - name: Terraform Apply Security Group Change
+        working-directory: src/test/resources/terraform
+        run: terraform apply --auto-approve
+
+      - name: Get ${{ matrix.database }}-${{ matrix.version }} Database Host URL
+        working-directory: src/test/resources/terraform
+        run: |
+          echo "DBENDPOINT=$(terraform output -json dbEndpoint | jq -r '."${{ matrix.database }}-${{ matrix.version }}"')" >> $GITHUB_ENV
+
+      - uses: liquibase/liquibase-github-action@v3
```

Comment:
What is this update doing that isn't handled by the test harness?

Reply:
These were copied over from the docker init scripts.  I assume they are creating objects that are required for the test-harness tests to run.  I just ran them using liquibase instead of a script.

Reply:
@mcred, yes the test-harness needs pre-populated tables with data for the tests to interact with. @jnewton03 took what we had in the init scripts and put them into a changelog run with Liquibase instead (https://github.com/liquibase/liquibase-test-harness/blob/DAT-7455/src/test/resources/terraform/changelogs/postgres.sql)